recipes/server_install.rb

```diff
@@ -51,23 +51,35 @@
     action :install
   end
 else
-  remote_file "#{Chef::Config[:file_cache_path]}/#{node['rundeck']['deb']['package']}" do
-    source node['rundeck']['url']
-    owner node['rundeck']['user']
-    group node['rundeck']['group']
-    checksum node['rundeck']['checksum']
-    mode '0644'
+  # bintray apt repo signed by bintray key and rundeck build key
+  # https://github.com/rundeck/rundeck/issues/93
+  # https://github.com/New-Edge-Engineering/ansible-rundeck/pull/17
+
+  apt_repository 'rundeck' do
+    uri node['rundeck']['deb']['repo']['url']
+    distribution '/'
+    key node['rundeck']['deb']['repo']['key']
+    action :add
+  end
+
+  # Hack to get multiple keys installed without borrowing apt_repository internals
+  # Abuse name property so we only write out one file in sources.list.d/
+  apt_repository 'rundeck-build' do
+    name 'rundeck'
+    uri node['rundeck']['deb']['repo']['url']
+    distribution '/'
+    key node['rundeck']['deb']['repo']['gpgkey']
+    action :add
   end
```

Comment:
This issue that requires this workaround is fixed in Chef 13.4.10, which supports an array of repo signing keys. We can possibly add a version check and if chef version `>= 13.4.10`, do the following:
```
apt_repository 'rundeck' do
   mode '0644' 
   uri node['rundeck']['deb']['repo']['url']
   distribution '/'
   key [node['rundeck']['deb']['repo']['key'], node['rundeck']['deb']['repo']['gpgkey']]
   action :add
 end
```
else do what is there just now.

13.4 isn't stable yet and we need to support 12 and earlier versions of 13.